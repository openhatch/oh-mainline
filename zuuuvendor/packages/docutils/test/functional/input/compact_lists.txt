* This is an ordinary simple bullet list.
* It should be made compact (<p> & </p> tags omitted).

**********

* This is a bullet list that is not simple.

  There are multiple paragraphs in some items.

* It should not be made compact.

* Even though some items may have only one paragraph.

**********

.. class:: open

* This is a simple bullet list, but class="open" is set.
* It should not be made compact.

**********

.. class:: compact

* This is a bullet list that is not simple.

  There are multiple paragraphs in some items.

* However, the class="compact" setting will cause 
  all first paragraph's <p> & </p> tags to be omitted.

* Items with multiple paragraphs will not appear changed.

* Items may have one paragraph, or multiple.

  Items with multiple paragraphs will still be followed
  by vertical whitespace because of the later paragraphs.

* The effect is interesting.
