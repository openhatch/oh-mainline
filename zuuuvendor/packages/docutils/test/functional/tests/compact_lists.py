# Source and destination file names.
test_source = "compact_lists.txt"
test_destination = "compact_lists.html"

# Keyword parameters passed to publish_file.
reader_name = "standalone"
parser_name = "rst"
writer_name = "html"

# Settings
# local copy of default stylesheet:
settings_overrides['stylesheet_path'] = ( 
    'functional/input/data/html4css1.css')
