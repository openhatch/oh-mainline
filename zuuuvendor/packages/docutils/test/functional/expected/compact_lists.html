<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.11: http://docutils.sourceforge.net/" />
<title></title>
<link rel="stylesheet" href="../input/data/html4css1.css" type="text/css" />
</head>
<body>
<div class="document">


<ul class="simple">
<li>This is an ordinary simple bullet list.</li>
<li>It should be made compact (&lt;p&gt; &amp; &lt;/p&gt; tags omitted).</li>
</ul>
<hr class="docutils" />
<ul>
<li><p class="first">This is a bullet list that is not simple.</p>
<p>There are multiple paragraphs in some items.</p>
</li>
<li><p class="first">It should not be made compact.</p>
</li>
<li><p class="first">Even though some items may have only one paragraph.</p>
</li>
</ul>
<hr class="docutils" />
<ul class="open">
<li><p class="first">This is a simple bullet list, but class=&quot;open&quot; is set.</p>
</li>
<li><p class="first">It should not be made compact.</p>
</li>
</ul>
<hr class="docutils" />
<ul class="compact simple">
<li>This is a bullet list that is not simple.<p>There are multiple paragraphs in some items.</p>
</li>
<li>However, the class=&quot;compact&quot; setting will cause
all first paragraph's &lt;p&gt; &amp; &lt;/p&gt; tags to be omitted.</li>
<li>Items with multiple paragraphs will not appear changed.</li>
<li>Items may have one paragraph, or multiple.<p>Items with multiple paragraphs will still be followed
by vertical whitespace because of the later paragraphs.</p>
</li>
<li>The effect is interesting.</li>
</ul>
</div>
</body>
</html>
